docs/content/labs/IPv6/hypervisor/redfish-for-vms.md

```diff
@@ -0,0 +1,16 @@
+In a bare metal environment, the preferred approach is to utilize the actual BMC (Baseboard Management Controller) of the nodes used for the management cluster, which can be managed by Metal3 for discovery and provisioning. However, in a virtual environment, this approach is not feasible. As a workaround, we will use `ksushy`, which is an implementation of `sushy-tools`, allowing us to simulate BMCs for the virtual machines.
```

Comment:
It would be nice to have a follow up PR that explains how to do it with boot it yourself